pkg/service/service.go

```diff
@@ -110,6 +110,54 @@ func DeleteServiceAndUnlinkComponents(client *occlient.Client, serviceName strin
 	return nil
 }
 
+// DeleteOperatorService deletes an Operator backed service
+// TODO: make it unlink the service from component as a part of
+// https://github.com/openshift/odo/issues/3563
+func DeleteOperatorService(client *kclient.Client, serviceName string) error {
+	serviceList, err := ListOperatorServices(client)
```

Comment:
Can a service in this list have the same name? If not, then I think we directly call `client.GetCSVWithCR(kind)` without listing. Or am I missing something?

Reply:
`GetCSVWithCR` returns the name of the CSV (Operator) which provides the kind (a Custom Resource a.k.a. CR). So, can't replace it here.